lte/cloud/go/services/subscriberdb/obsidian/models/swagger.v1.yml

```diff
@@ -505,6 +505,7 @@ definitions:
       last_reported_time:
```

Comment:
If possible please add description field to `last_reported_time` property (idk if Swagger will let you?)